RickAndMorty/Models/RMEpisode.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RMEpisode.swift
+//  RickAndMorty
+//
+//  Created by Juan Camilo Fonseca Gomez on 11/07/23.
+//
+
+import Foundation
+
+struct RMEpisode: Decodable {
+    var id: Int
+    var name: String
+    var air_date: String
+    var episode: String
+    var characters: [String]
```

Comment:
Are we sure this characters are `String` ?

Reply:
This is getting a url string of character.
Can you add the extension for fetching the Id that we add to your repo and convert this `characters` to an array of Int ?

Reply:
https://github.com/SSPF20/RickAndMortyAPI/blob/11e9a8d0f3bb7657cabd0f8290ad19aecc6454b5/RickAndMorty/Extensions/String%2BExt.swift
We already have the extension for that